db/migrations/03_create_test_tables.sql

```diff
@@ -0,0 +1,4 @@
+-- testdb --
```

Comment:
Do not need this file. If notes for creating database are in readme this will be handles by 01_create_tables file

Reply:
Ty ty done